frontend/src/App.tsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import Navbar from "./components/navbar/Navbar";
+import Main from "./pages/main/Main";
+
+function App() {
+  const [darkMode, setDarkMode] = useState<boolean>(false);
+
+  const toggleDarkMode = () => {
+    setDarkMode(!darkMode);
+    console.log(darkMode);
```

Comment:
preferably no debugging console.logs in production code (same reason as comments)

Reply:
resolved